src/bodywork/cli/workflow_jobs.py

```diff
@@ -75,7 +74,7 @@ def delete_workflow_job_in_namespace(namespace: str, job_name: str) -> None:
 
 def create_workflow_cronjob(
```

Comment:
Is it worth keeping `namespace` as an arg here, given that it's an arg in every other function in this module and then setting it to `BODYWORK_DEPLOYMENT_JOBS_NAMESPACE` in `cli.py`?

Reply:
Yes, I it's more consistent.